src/metabase/models/user.clj

```diff
@@ -100,7 +100,7 @@
         (db/insert! PermissionsGroupMembership
           :group_id (u/the-id (perms-group/admin))
           :user_id  id)
-        ;; don't use [[db/delete!]] here because that does the opposite and tries to update this user which leads to a
+        ;; don't use [[t2/delete!]] here because that does the opposite and tries to update this user which leads to a
         ;; stack overflow of calls between the two. TODO - could we fix this issue by using a `post-delete` method?
```

Comment:
hmm, this can probably be fixed properly now

Reply:
ah yea that's on my todo list for converting `simple-x` stuffs. basically, you will call `db/delete!` directly with the table name, not a model